<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>TGo4OpenRemoteDialog</class>
  <widget class="QDialog" name="TGo4OpenRemoteDialog" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>304</width>
        <height>139</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Open remote root file:</string>
    </property>
    <property name="sizeGripEnabled" >
      <bool>true</bool>
    </property>
    <layout class="QGridLayout" >
      <item row="0" column="0" >
        <layout class="QHBoxLayout" >
          <item>
            <layout class="QVBoxLayout" >
              <item>
                <widget class="QLabel" name="ServerLabel" >
                  <property name="text" >
                    <string>Server:</string>
                  </property>
                  <property name="wordWrap" >
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="NameLabel" >
                  <property name="text" >
                    <string>File Name:</string>
                  </property>
                  <property name="wordWrap" >
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="TypeLabel" >
                  <property name="text" >
                    <string>File Type:</string>
                  </property>
                  <property name="wordWrap" >
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <layout class="QVBoxLayout" >
              <item>
                <widget class="QLineEdit" name="ServerEdit" >
                  <property name="toolTip" stdset="0" >
                    <string>Remote server node name</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLineEdit" name="FileNameEdit" >
                  <property name="toolTip" stdset="0" >
                    <string>Filename without url to remote server</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QComboBox" name="TypeSelectorBox" >
                  <property name="toolTip" stdset="0" >
                    <string>Type of remote connection</string>
                  </property>
                  <item>
                    <property name="text" >
                      <string>TNetFile/XRootd (root:)</string>
                    </property>
                  </item>
                  <item>
                    <property name="text" >
                      <string>RFIO (rfio:)</string>
                    </property>
                  </item>
                  <item>
                    <property name="text" >
                      <string>WebFile (http:)</string>
                    </property>
                  </item>
                  <item>
                    <property name="text" >
                      <string>Local file</string>
                    </property>
                  </item>
                </widget>
              </item>
            </layout>
          </item>
        </layout>
      </item>
      <item row="1" column="0" >
        <layout class="QHBoxLayout" >
          <item>
            <spacer name="spacer35" >
              <property name="sizeHint" >
                <size>
                  <width>191</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="buttonCancel" >
              <property name="text" >
                <string>&amp;Cancel</string>
              </property>
              <property name="shortcut" >
                <string/>
              </property>
              <property name="autoDefault" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="buttonOk" >
              <property name="text" >
                <string>&amp;OK</string>
              </property>
              <property name="shortcut" >
                <string/>
              </property>
              <property name="autoDefault" >
                <bool>true</bool>
              </property>
              <property name="default" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <connections>
    <connection>
      <sender>buttonOk</sender>
      <signal>clicked()</signal>
      <receiver>TGo4OpenRemoteDialog</receiver>
      <slot>accept()</slot>
    </connection>
    <connection>
      <sender>buttonCancel</sender>
      <signal>clicked()</signal>
      <receiver>TGo4OpenRemoteDialog</receiver>
      <slot>reject()</slot>
    </connection>
    <connection>
      <sender>TypeSelectorBox</sender>
      <signal>activated(QString)</signal>
      <receiver>TGo4OpenRemoteDialog</receiver>
      <slot>TypeSelectorBox_activated(QString)</slot>
    </connection>
    <connection>
      <sender>ServerEdit</sender>
      <signal>textChanged(QString)</signal>
      <receiver>TGo4OpenRemoteDialog</receiver>
      <slot>ServerEdit_textChanged(QString)</slot>
    </connection>
    <connection>
      <sender>FileNameEdit</sender>
      <signal>textChanged(QString)</signal>
      <receiver>TGo4OpenRemoteDialog</receiver>
      <slot>FileNameEdit_textChanged(QString)</slot>
    </connection>
  </connections>
</ui>
